docusaurus.config.js

```diff
@@ -101,43 +121,58 @@ const config = {
             title: 'Docs',
             items: [
               {
-                label: 'Documentation',
+                label: 'Lisk Documentation',
                 to: '/',
               },
             ],
           },
           {
             title: 'Community',
             items: [
-              /* {
-                 label: 'Stack Overflow',
-                 href: 'https://stackoverflow.com/questions/tagged/docusaurus',
-               },
-               {
-                 label: 'Discord',
-                 href: 'https://discordapp.com/invite/docusaurus',
-               },
-               {
-                 label: 'Twitter',
-                 href: 'https://twitter.com/docusaurus',
-               },*/
+              {
+                html: `
+                    <a href="https://discord.com/invite/7EKWJ7b" target="_blank" aria-label="Discord">
+                    <img src="img/icon-discord-white.svg" class="git-dark" />
+                    <img src="img/icon-discord.svg" class="git-light" />
+                    </a>
```

Comment:
Maybe it looks better with the name behind the icon?

Same for GitHub and Twitter links.

```suggestion
                    Discord
                    </a>
```

Reply:
Tried it, doesn't look good! I think it should be without icons altogether in the footer, like it was in the default setting.

Reply:
Imho it will make it clearer with text, like we had it in the footer of our [old docs](https://lisk.com/documentation/). What exactly do you mean by it doesn't look good?

Reply:
<img width="170" alt="image" src="https://github.com/LiskHQ/lisk-documentation/assets/13951043/132dca21-7c65-4de1-adbd-4e260312dd0b">

I can align the text and the icon with styling but still, I prefer the default links over this one!

Reply:
Hmmm ok, then if you prefer only icons, maybe it looks better if they are in the same line

Reply:
I have now styled them to be on the same line!